samples/sample_hevc_fei/include/ref_list_manager.h

```diff
@@ -129,10 +130,20 @@ class EncodeOrderControl
 
     HevcTask* GetTask(mfxFrameSurface1 * surface)
     {
-        HevcTask* task = ReorderFrame(surface);
-        if (task)
+        HevcTask* task = NULL;
+        if (m_reorder)
```

Comment:
I don't like unnecessary renaming. Also now class called 'Reorder' can be not a reorder.

Reply:
The same for EncodeOrderControl - it can be not encode order.